implementations/index.md

```diff
@@ -95,6 +95,7 @@ isomorphic ActiveRecord clone that issues JSON:API requests instead of SQL and i
 * [enm/json-api-client](https://eosnewmedia.github.io/JSON-API-Client/) is an abstract client-side PHP implementation of the json:api specification which is based on [enm/json-api-common](https://eosnewmedia.github.io/JSON-API-Common/). It allows you to send json:api requests via your own http client implementation or via a buzz or guzzle client.
 * [floor9design-ltd/json-api-formatter](https://github.com/floor9design-ltd/json-api-formatter) An open source library designed to be used for generating and interacting with (making and receiving) JSON API requests. 100% unit tested and actively maintained by the author. Easy to use with frameworks such as Laravel/Symfony. 
 * [swisnl/json-api-client](https://github.com/swisnl/json-api-client) Is a package for mapping remote {json:api} resources to Eloquent like models and collections.
+* [API Platform](https://api-platform.com) is an API-first Framework supporting JSON:API. 
```

Comment:
Shouldn't this go into server-libraries?

Reply:
oh my bad, updated